nixos/modules/profiles/lxc-container.nix

```diff
@@ -0,0 +1,47 @@
+{ config, pkgs, lib, ... }: {
+
+  boot.isContainer = true;
+
+  # TODO: build rootfs as squashfs for faster unpack
+  system.build.tarball = pkgs.callPackage ../../lib/make-system-tarball.nix {
+    extraArgs = "--owner=0";
+
+    storeContents = [
+      {
+        object = config.system.build.toplevel;
+        symlink = "none";
+      }
+    ];
+
+    contents = [
+      {
+        source = config.system.build.toplevel + "/init";
+        target = "/sbin/init";
+      }
+    ];
+
+    extraCommands = "mkdir -p proc sys dev";
+  };
+
+  # Add the overrides from lxd distrobuilder
+  systemd.extraConfig = ''
```

Comment:
Please update overrides:
https://github.com/lxc/distrobuilder/blob/f15eec09df7b04f1bede66b0f31354da66748c9a/distrobuilder/main.go#L622

Reply:
Done, thanks.

Any idea what the purpose of the empty `LoadCredential=` is?

Reply:
An empty directive will delete the directive during override.

Reply:
Ah ok, I understand.

Reply:
This block actually isn't serving its purpose. It was already incorrect, but updating it isn't making a difference.

```
Aug 27 00:33:34 wire1 systemd[1]: /etc/systemd/system.conf:14: Unknown section 'Service'. Ignoring.
```

These need to be service configs, added to the services. Distrobuilder either adds it to `/etc/systemd/system/service.d/lxc.conf` or every .service file in systemd's path. https://github.com/lxc/distrobuilder/blob/f15eec09df7b04f1bede66b0f31354da66748c9a/distrobuilder/main.go#L713-L718

I don't see a way to add it to `service.d` currently, but am happy to be corrected. The original author actually [asked about creating this global service override](https://github.com/NixOS/nixpkgs/pull/120965#issuecomment-828113362), but it ended up here incorrectly.